app/src/main/java/com/example/lemonade/ui/theme/Theme.kt

```diff
@@ -126,3 +126,23 @@ fun AppTheme(
         content = content
     )
 }
+
+/**
+ * Sets up edge-to-edge for the window of this [view]. The system icon colors are set to either
+ * light or dark depending on whether the [darkTheme] is enabled or not.
+ */
+private fun setUpEdgeToEdge(view: View, darkTheme: Boolean) {
+    val window = (view.context as Activity).window
+    WindowCompat.setDecorFitsSystemWindows(window, false)
+    window.statusBarColor = Color.Transparent.toArgb()
+    val navigationBarColor = when {
+//        Build.VERSION.SDK_INT >= 29 -> Color.Transparent.toArgb()
```

Comment:
Why is `Color.Transparent.toArgb()` not working for SDK version 29 and higher?

Reply:
What do you mean?

On API 29+, the nav bar gets automatic scrim if the device is set to 3-button nav. Is this what you mean?

Reply:
Thank you